v3.0/tidb-in-kubernetes/maintain/backup-and-restore.md

```diff
@@ -128,7 +128,7 @@ Incremental backup uses [TiDB Binlog](/v3.0/reference/tidb-binlog-overview.md) t
 
 For the detailed guide of maintaining TiDB Binlog in Kubernetes, refer to [TiDB Binlog](/v3.0/tidb-in-kubernetes/maintain/tidb-binlog.md).
 
-### Decrease the capability of Pump
+### Scale Pump in
 
 To decrease the capability of Pump, make Pump nodes offline and then run the `helm upgrade` command to delete Pump pods.
```

Comment:
`To decrease the capability of Pump`

this should be modified

Reply:
Got it.